src/data/fonts.js

```diff
@@ -0,0 +1,705 @@
+export const top250 = [
```

Comment:
This is some real crazy file.
At least this is lazy loaded we may find a better way to split if with an ovevrride of webpack confif but for now this ok

Reply:
Agree, it's this or loading the 1500 fonts so... And right now it's the only possibility from the remotion doc